roles/openshift_node_upgrade/tasks/registry_auth.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Check for credentials file for registry auth
+  stat:
+    path: "{{ oreg_auth_credentials_path }}"
```

Comment:
So this file is now duplicated across the `openshift_node` and `openshift_node_upgrade` roles. Not very DRY. Any way around that?

Reply:
Next cycle I would like to see openshift_node and openshift_node_upgrade roles combined into one.  We could use include_role.  That will remove lots of duplicated items and work-arounds (like the set-facts I have to do in openshift_master).  Same applies to openshift_master.